core/src/PrognosticData.cpp

```diff
@@ -177,13 +194,29 @@ ModelState PrognosticData::getStateRecursive(const OutputSpec& os) const
     return os ? state : ModelState();
 }
 
-PrognosticData::HelpMap& PrognosticData::getHelpText(HelpMap& map, bool getAll) { return map; }
+PrognosticData::HelpMap& PrognosticData::getHelpText(HelpMap& map, bool getAll)
+{
+
+    map["Debug"] = {
+        { fieldNamesKey, ConfigType::STRING, {}, fieldNamesDefault, "",
+            "Comma separated, space free list of fields to be checked if check_fields is true. "
+            "The special value \""
+                + all + "\" will output all available fields." },
+        { checkFieldsKey, ConfigType::BOOLEAN, { "true", "false" },
+            ConfigurationHelp::toString(checkFieldsDefault), "",
+            "Switch to check if the fields listed in field_names are within a reasonable "
+            "physical range and not NaN." },
+    };
+    return map;
+}
+
```

Comment:
Put the common help test pertaining to checking into the `CheckingModelComponent` class.